File: onebot/src/main/java/cn/evole/onebot/sdk/event/notice/friend/PrivatePokeNoticeEvent.java
package cn.evole.onebot.sdk.event.notice.friend;

import cn.evole.onebot.sdk.event.notice.NoticeEvent;
import com.google.gson.annotations.SerializedName;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

/**
 * Created on 2022/7/8.
 *
 * @author cnlimiter
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
@EqualsAndHashCode(callSuper = true)
public class PrivatePokeNoticeEvent extends NoticeEvent {

    @SerializedName("sub_type")
    public String subType;

    @SerializedName("sender_id")
    public long senderId;

    @SerializedName("target_id")
    public long targetId;



}

File: onebot/src/main/kotlin/sdk/event/notice/friend/PrivatePokeNoticeEvent.kt
package cn.evolvefield.onebot.sdk.event.notice.friend

import cn.evolvefield.onebot.sdk.event.notice.NoticeEvent
import com.google.gson.annotations.SerializedName
import lombok.AllArgsConstructor
import lombok.Data
import lombok.EqualsAndHashCode
import lombok.NoArgsConstructor

@Data
@NoArgsConstructor
@AllArgsConstructor
@EqualsAndHashCode(callSuper = true)
class PrivatePokeNoticeEvent : NoticeEvent() {
    @SerializedName("sub_type")
    var subType = ""
    @SerializedName("sender_id")
    var senderId = 0L
    @SerializedName("target_id")
    var targetId = 0L
}
